Removed section that finds jasper.
It is not needed since we use openjpeg instead, but the CMakeLists.txt inadvertently forces looking for jasper.

--- CMakeLists.txt	2019-07-08 14:06:44.000000000 +0200
+++ CMakeLists.txt	2019-07-19 13:27:24.158310812 +0200
@@ -196,11 +196,6 @@
     #       which can affect future package discovery if not undone by the caller.
     #       The current CMAKE_PREFIX_PATH is backed up as _CMAKE_PREFIX_PATH
     #
-    set(CMAKE_WARN_DEPRECATED OFF) # Suppress deprecation message
-    ecbuild_add_extra_search_paths( jasper )
-    find_package( Jasper )
-    set(CMAKE_PREFIX_PATH ${_CMAKE_PREFIX_PATH})    # Restore CMAKE_PREFIX_PATH
-    set(CMAKE_WARN_DEPRECATED ON)  # Remove suppression
 
     find_package( OpenJPEG )
 
